interpreter/cling/lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -43,38 +41,29 @@ namespace cling {
     // Behaviour is to not add paths that don't exist...In an interpreted env
     // does this make sense? Path could pop into existance at any time.
     for (const char* Var : kSysLibraryEnv) {
-      if (Opts.Verbose())
-        cling::log() << "Adding library paths from '" << Var << "':\n";
       if (const char* Env = ::getenv(Var)) {
         llvm::SmallVector<llvm::StringRef, 10> CurPaths;
-        SplitPaths(Env, CurPaths, utils::kPruneNonExistant, platform::kEnvDelim,
-                   Opts.Verbose());
+        SplitPaths(Env, CurPaths, utils::kPruneNonExistant, platform::kEnvDelim);
         for (const auto& Path : CurPaths)
-          m_SearchPaths.push_back({Path.str(), /*IsUser*/true});
+          addSearchPath(Path.str());
       }
     }
 
+    // This will currently be the last path searched, should it be pushed to
```

Comment:
This comment doesn't seem to be correct anymore - after this PR, system search paths come last. Unless there's a good reason for this (I didn't find this mentioned in the commit logs), maybe revert the order to have "." last again? Otherwise please adjust the comment.

Reply:
Fixed the comment. The intent the current folder to be a last resort.

Reply:
But then "." should be after the system paths, and the comment should have stayed? My confused :-)

Reply:
Ok, misspoke. Should be right after the user-defined library paths as this is an implicit one. Making it a path considered after the system libraries will likely cause a huge performance regression for not really good reason...

Reply:
> will likely cause a huge performance regression

"Regression"? Note that "." is the last dir entry (after system dirs) before this PR. And as system libraries are used quite regularly, and as anything under the home directory might be network paths (AFS!), I'm not sure moving "." before system paths will *improve* performance?

Reply:
As I do not forget many of the details, I can only say that the old state of the PR made your test (above) fail.

Reply:
> As I do not forget many of the details

Lucky you :-) OK then, let's see!